### DISCUSSION TITLE
What is the difference of `string?` and `stringlike?` ?

### DISCUSSION THREAD

**Discussion by ghost** [h3rald/min#91] [Q&A] [answered]
I used `string?` and `stringlike?` at some examples:
```
"abc" string? puts ;true
 123   string? puts ;false

"abc" stringlike? puts ;true
 123   stringlike? puts ;false
```
They seem same. What is the difference of them?

**Comment by h3rald** [comment, marked as answer]
`stringlike?` returns true for *quoted symbols* as well as strings:

     'test stringlike? ;true

Quoted symbols are commonly used kind of interchangeably with strings, in some cases.

**Reply by ghost** [reply]
Okey! Is understood 👍
